src/coreclr/src/tools/aot/jitinterface/corinfoexception.h

```diff
@@ -9,17 +9,17 @@ typedef char16_t WCHAR;
 typedef wchar_t WCHAR;
 #endif
 
-class CorInfoException
```

Comment:
question: why was it renamed?

Reply:
CorInfoException is an enum type in corinfo.h